views/update/update-beneficial-owner-type.html

```diff
@@ -94,7 +94,10 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ title }}</h1>
     <form method="post">
       {% include "includes/csrf_token.html" %}
       {% include "includes/beneficial-owner-type-add-radios.html" %}
-
+      {% if update.change_bo_relevant_period === "CHANGE_BO_RELEVANT_PERIOD" %}
```

Comment:
Can you add `feature_flag_relevant_period === true` to this if statement to protect from unwanted behavior